src/core/filters/filterregistry.js

```diff
@@ -76,6 +76,16 @@ export default class FilterRegistry {
     return Facet.fromFilters(this.availableFieldIds, ...filters);
   }
 
+  /**
+   * Gets the locationRadius to send in a search query.
+   * @returns {number}
+   */
+  getLocationRadiusPayload () {
+    const filterNode =
+      this.globalStorage.getState(`${this.filterRegistryId}.${StorageKeys.LOCATION_RADIUS}`);
```

Comment:
As we talked about last night, I think we should remove filterRegistryId as it's no longer being used. For the sake of separation of concerns, I think we should do that in a separate PR.

Reply:
got it